projects/plugins/jetpack/modules/wordads/php/class-wordads-smart.php

```diff
@@ -207,10 +205,13 @@ private function get_config_url(): string {
 	/**
 	 * Places marker at the end of the content so inline can identify the post content container.
 	 *
-	 * @param string $content The post content.
-	 * @return string The post content with the marker appended.
+	 * @param string|null $content The post content.
+	 * @return string|null The post content with the marker appended.
 	 */
-	private function insert_inline_marker( string $content ): string {
+	private function insert_inline_marker( $content ) {
```

Comment:
This function needs to change to `public` now

Reply:
Thanks! 7ee70666ba

Reply:
Looks great!

I see ads and no errors.

![Uploading Screenshot 2024-05-14 at 10.21.14 AM.png…]()